test/test_torch.py

```diff
@@ -12457,8 +12457,45 @@ def test_has_internal_overlap(self):
     @staticmethod
     def unary_check_mem_overlap(self, inplace_op, value=-0.5, device='cpu'):
         tensor = torch.tensor(value, device=device).expand(3, 3)
-        with self.assertRaisesRegex(RuntimeError, 'single memory location'):
-            inplace_op(tensor)
+        if device == 'cuda' and inplace_op == torch.Tensor.sigmoid_:
```

Comment:
This should really not be special-cased here. You should add a new test for this and mark it as an expected failure:

```
@unittest.expectedFailure
def sigmoid_mem_overlap_cuda(self):
    unary_check_mem_overlap(...)
```

Reply:
addressed